demisto_sdk/commands/validate/validators/PB_validators/PB125_playbook_only_default_next.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+class PlaybookOnlyDefaultNextValidator(BaseValidator[ContentTypes]):
+    error_code = "PB125"
+    description = (
+        "Validates that a condition task doesn't has only a default next-task."
+    )
+    rationale = "Validates that a condition task doesn't has only a default next-task."
+    error_message = (
+        'Playbook with id:"{playbook_id}" has conditional tasks with an only default condition. Tasks IDs: {tasks}.\n'
```

Comment:
@YuvHayun do we need to specify the playbook ID in the error message?

Reply:
Nope, the file path will appear as a part of the message.

Reply:
removed